packages/data-service/package.json

```diff
@@ -46,7 +46,7 @@
     "lint": "npm run eslint . && npm run prettier -- --check .",
     "check": "npm run lint && npm run depcheck",
     "check-ci": "npm run check",
-    "pretest": "mongodb-runner start --port=27018",
+    "pretest": "mongodb-runner start --enterprise --mongodb-version '>= 6.0.0-rc0' --topology=replicaset --secondaries=0 --port=27018",
```

Comment:
This requires the mongodb-runner version bump from https://github.com/mongodb-js/compass/pull/3016.

Is this okay? It’s a bit odd for sure, but it’s also not really bad per se?

(The replicaset config comes from the fact that FLE2 is only allowed with replica sets.)